### DISCUSSION TITLE
Idiomatic way of saving files

### DISCUSSION THREAD

**Discussion by CodeAssemblingChicken** [bevyengine/bevy#5937] [Q&A] [answered]
For a game, you often need to save settings, profiles or similar.
Does bevy provide a 'proper' way of doing so? While rust's std::fs::File::create does work on native platforms, it's no use on web builds.
I think #1442 relates to this, has anything changed in that department? 

Thanks for your help.

**Comment by bjorn3** [comment, marked as answer]
I think you will have to use the web-sys crate to access localStorage.

**Reply by CodeAssemblingChicken** [reply]
OK if bevy does not yet provide this feature I'll look into it.
Thank you for the quick reply
